site/docs/v1/tech/apis/identity-providers/samlv2-idp-initiated.adoc

```diff
@@ -0,0 +1,213 @@
+---
+layout: doc
+title: SAML v2 IdP Initiated Identity Provider APIs
+description: APIs for creating, retrieving, updating and deleting SAML v2 IdP Initiated identity providers
+---
+
+== Overview
+
+[NOTE.since]
+====
+This API has been available since 1.26.0
+====
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+The SAML v2 IdP Initiated IdP initiated Identity Provider allows an external IdP to send an unsolicited `AuthN` request when FusionAuth is acting as the Service Provider, or SP.
```

Comment:
```suggestion
The SAML v2 IdP Initiated IdP initiated Identity Provider allows an external IdP to send an unsolicited `AuthN` request when FusionAuth is acting as the Service Provider (or SP).
```
I think putting the acronym in parenthesis reads better?  We do structure it this way on "docs/v1/tech/identity-providers/samlv2-idp-initiated/"